lde_config/lde/node/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:18
+WORKDIR /srv/api/aiproxy
+
+# If you are building your code for production
+# RUN npm ci --omit=dev
+# Bundle app source
+COPY ../../.. .
```

Comment:
This is not needed for LDE, as the docker-compose file mounts the current dev code. But it also doesn't hurt.

Reply:
I'm surprised by the `../../..` bit, as the build context in the docker-compose file is `.`  from the root https://github.com/klauste/mockai/pull/1/files#diff-05e39158df47d1d49074fef2681a1b22afa5387a51e830d708613c64b63ad0ceR18-R20